cmd/fleetctl/get.go

```diff
@@ -1022,3 +1055,55 @@ func getSoftwareCommand() *cli.Command {
 		},
 	}
 }
+
+func getMDMAppleCommand() *cli.Command {
+	return &cli.Command{
+		Name:    "mdm_apple",
+		Hidden:  true, // TODO: temporary, until the MDM feature is officially released
+		Aliases: []string{"mdm-apple"},
+		Usage:   "Show Apple Push Notification Service (APNs) information",
+		Flags: []cli.Flag{
+			configFlag(),
+			contextFlag(),
+			debugFlag(),
+		},
+		Action: func(c *cli.Context) error {
+			const expirationWarning = 30 * 24 * time.Hour // 30 days
+
+			client, err := clientFromCLI(c)
+			if err != nil {
+				return err
+			}
+
+			mdm, err := client.GetAppleMDM()
+			if err != nil {
+				var nfe service.NotFoundErr
+				if errors.As(err, &nfe) {
+					log(c, "Error: No Apple Push Notification service (APNs) certificate found. Use `fleetctl generate mdm-apple` and then `fleet serve` with `mdm` configuration to turn on MDM features.\n")
```

Comment:
leaving a note that this will also print this message if a new `fleetctl` talks to an old `fleet`, but I personally think it's OK.

Reply:
That's a good point, but then the message printed is still true.